src/data-loaders/RelationalDataLoader.js

```diff
@@ -0,0 +1,64 @@
+import Sequelize from 'sequelize';
+import get from 'lodash/get'
+import Promise from 'bluebird'
+
+export default class RelationalDataLoader {
+  constructor(config) {
+    this.config = config;
+  }
+
+  async load(req) {
+    try {
+      const requiredKeys = [
+        'dbDialect',
+        'dbUsername',
+        'dbPassword',
+        'dbHost',
+        'dbName',
+      ];
+      if (!this.config.rds) {
+        throw new Error('RDS configuration not passed');
+      }
+      const missingKey = requiredKeys.find((key) => {
+        return !this.config.rds[key];
+      });
+      if (missingKey) {
+        throw new Error(`${missingKey} is required.`);
+      }
+      const config = {
+        uri: `${this.config.rds.dbDialect}://${this.config.rds.dbUsername}:${this.config.rds.dbPassword}@${this.config.rds.dbHost}:${this.config.rds.dbPort}/${this.config.rds.dbName}`,
+        options: {
+          host: process.env.DB_HOST,
+          logging: console.log,
+          dialect: process.env.DB_DIALECT,
```

Comment:
@alichherawalla What is the difference between these `options` and what is in the `uri`?

Should this use values from `this.config.rds`?